### DISCUSSION TITLE
How to generate HTTPS share links

### DISCUSSION THREAD

**Discussion by dibakarb** [photoprism/photoprism#3527] [Q&A] [answered]
Hello,

How can I create share HTTPS links instead of HTTP. I installed PhotoPrism as an OpenMediaVault plugin but guess that doesn't make difference for link generation.

Thanks,
Diba

**Comment by ippocratis** [comment]
A pupblic domain / dynamic dns subdomain and reverse proxy that with lets encrypt cert for so that you can use tls?
e.g. caddy v2
https://docs.photoprism.app/getting-started/proxies/caddy-2/

**Comment by dibakarb** [comment]
My install is working fine as an HTTPS url, like - https://mywebsite.com:2053/library/folders. But when I am clicking on share button it is creating http url with a different port number, like - http://mywebsite.me:2342/s/3atj25ae1j/sms-moments. So what do I need to do so that it generates correct link.

Edit: Corrected typo.

**Reply by lastzero** [reply]
Change the site URL in the config options as shown in our docs and examples.

**Reply by dibakarb** [reply]
Thanks.

I am not familiar with this technology stack so forgive me if my questions are too basic. Looks like I have to update PHOTOPRISM_SITE_URL in docker-compose.yml, am I correct? What if I installed without docker through third party, like Openmediavault, the docs are silent on this.

Ref: [https://docs.photoprism.app/getting-started/config-options/](https://docs.photoprism.app/getting-started/config-options/)

Edit: Corrected reference url.

**Reply by lastzero** [reply]
Our docs are silent because it's a third party integration. There should be a way to set the variable. Note we are on vacation and can't help with this right now.

**Reply by dibakarb** [reply]
Thanks, I guess I have enough information to ask in omv forum.

**Comment by dibakarb** [comment, marked as answer]
Solution - [https://github.com/openmediavault/openmediavault/issues/1372](https://github.com/openmediavault/openmediavault/issues/1372)
